setup.sh

```diff
@@ -75,6 +75,21 @@ function greenprint {
     echo -e "\033[1;32m${1}\033[0m"
 }
 
+# Check file permission
+check_key_perm () {
+    KEY_FILE=$1
+    KEY_PERM=$(stat -L -c "%a %G %U" $1 | grep -oP '\d+' | head -n 1)
+    echo -e "${KEY_PERM}"
+}
+
+# Check ostree_key permissions
+KEY_PERMISSION_PRE=$(check_key_perm "./key/ostree_key")
```

Comment:
Is that possible to just use one command instead of a function? Like
KEY_PERMISSION_PRE=$(stat -L -c "%a %G %U" key/ostree_key | grep -oP '\d+' | head -n 1)
That will use less code but have same result.